### DISCUSSION TITLE
Welcome to millionaire Discussions!

### DISCUSSION THREAD

**Discussion by Eng1Mahmoud** [Eng1Mahmoud/millionaire#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nascase** [comment]
I will like to get more ideas on

**Reply by Rilwanu97** [reply]
Hello sir

**Reply by Nascase** [reply]
U are welcome
